guestbook/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import include, url
+from django.http import HttpResponseRedirect
+from django.core.urlresolvers import reverse
+
+urlpatterns = [
+    url(r'^$', lambda request: HttpResponseRedirect(reverse('mypage'))),
```

Comment:
Cute trick... you could also have used the [`redirect`](https://docs.djangoproject.com/en/1.8/topics/http/shortcuts/#redirect) shortcut:

    from django.shortcuts import redirect

    ...

    url(r'^$', lambda request: return redirect('mypage')),

There is also the [`RedirectView`](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#django.views.generic.base.RedirectView):

    from django.views.generic import RedirectView

    url(r'^$', RedirectView.as_view(pattern_name='mypage')),